### PR TITLE
[Keyboard] Add support for Infinity87 Rev. 3

### PR DESCRIPTION
**Pull request by mechlovin**
## Description
Add support for Infinity87 Rev. 3, APM32F103
Thanks.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mechlovin/infinity87/rev3/standard/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# infinity87
+
+![infinity87](https://i.imgur.com/QuK1EnNl.png)
+
+A replacement PCB for TKL keyboard. A87 form factor,south-facing stabilisers.  
+
+* Keyboard Maintainer: [Team Mechlovin'](https://github.com/mechlovin)
```
[the remark this answers is not in the document]
Comment (mechlovin) [reply keyboards/mechlovin/infinity87/rev3/standard/readme.md:7]: Thanks, I will change to "Mechlovin' Studio"

**Review by drashna — APPROVED** [review]

File: keyboards/mechlovin/infinity87/rev3/rouge87/info.json [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    "matrix_pins": {
+        "rows": ["A10", "B13", "B12", "B11", "C14", "C15"],
+        "cols": ["C13", "B9", "B4", "B7", "B8", "B5", "B6", "A9", "A5", "A6", "A7", "B1", "B2", "B10", "B3", "B14", "B15"]
+    },
+    "indicators": {
+        "caps_lock": "A4",
+        "scroll_lock": "A8"
+    },
+    "layouts": {
+        "LAYOUT_ansi": {
```
[the remark this answers is not in the document]
Comment (mechlovin) [reply keyboards/mechlovin/infinity87/rev3/rouge87/info.json:15]: Hi, the "2N" key does not exist, here is the KLE for Infinity87 Rev. 3: 
http://www.keyboard-layout-editor.com/#/gists/745c76d01db56d915b00330d6c2f4900
Thank you.

**Review by fauxpark — CHANGES_REQUESTED** [review]
Keyboard updates need to go through `develop`.

File: keyboards/splitkb/aurora/sofle_v2/rev1/config.h [hunk 3]
```diff
@@ -1,4 +1,12 @@
+<<<<<<<< HEAD:keyboards/splitkb/aurora/sofle_v2/rev1/config.h
```
Comment (drashna) [inline keyboards/splitkb/aurora/sofle_v2/rev1/config.h:1]: merge conflists

File: keyboards/mechlovin/infinity87/rev3/standard/config.h [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define WS2812_SPI SPID2
```
Comment (drashna) [inline keyboards/mechlovin/infinity87/rev3/standard/config.h:19]:
```suggestion
#define WS2812_SPI_DRIVER SPID2
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**